src/main/java/io/stargate/sgv2/jsonapi/exception/mappers/ThrowableToErrorMapper.java

```diff
@@ -120,6 +121,11 @@ public final class ThrowableToErrorMapper {
               .getCommandResultError(message, Response.Status.INTERNAL_SERVER_ERROR);
           // Unidentified Driver Exceptions, will not map into JsonApiException
         } else if (throwable instanceof DriverException) {
+          if (throwable instanceof ReadFailureException) {
+            return ErrorCode.DATABASE_READ_FAILED
+                .toApiException()
```

Comment:
I think we should include actual underlying error message for extra information: something like:

    .toApiException("root cause: (%s) %s", throwable.getClass().getName(), throwable.getMessage())